red: ["mcamarneiro","jxavier","idomingues"]
green: ["testeves","jleao","jsantos"]
blue: ["mriem","dcampos","mmiranda"]

hunting_distance: 0.5
game_duration: 120
positive_score: 1
negative_score: -1





